src/renderer/components/_molecules/data-types/enumerated/index.tsx

```
@@ -50,24 +149,70 @@ const EnumeratorDataType = ({ data, ...rest }: EnumDatatypeProps) => {
+                      <SelectItem
+                        key={value.description}
+                        value={value.description}
+                        className='flex w-full cursor-pointer items-center justify-center py-1 outline-none hover:bg-neutral-100 dark:hover:bg-neutral-800'
+                        onClick={() => handleInitialValueChange(value.description)}
+                      >
+                        <span className='text-center font-caption text-xs font-normal text-neutral-700 dark:text-neutral-100'>
+                          {value.description}
+                        </span>
+                      </SelectItem>
+                    )}
+                  </>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)